src/dns.rs

```diff
@@ -63,8 +63,8 @@ impl fmt::Display for ResolveOneCtx {
 
 impl Resolver {
 
-    /// Construct a new `Resolver` from the system configuration and Conduit's
-    /// environment variables.
+    /// Construct a new `Resolver` from the system configuration and the
+    /// proxy's environment variables.
```

Comment:
I would just remove "the proxy's" too. After all, who else are we going to get environment variables from, other than what's available in our own process?